packages/hardhat-core/src/internal/core/errors-list.ts

```diff
@@ -201,6 +201,19 @@ Please run: npm install --save-dev typescript`,
 Please run this and try again: \`npm install --save-dev typescript\``,
       shouldBeReported: false,
     },
+    INVALID_PRIVATE_KEY: {
```

Comment:
I think that instead of adding an error type i'd still use invalid config, but making sure that the error message is clear. @fvictorio, is this what you had in mind?

Reply:
Yes. Something similar to the other "manual" checks we do for the config that are not part of `io-ts` validation.

Reply:
Got it :+1: